notifications/defaultPreferences.ts

```diff
@@ -1,18 +1,28 @@
-import { NotificationPreferences } from '../graphql/types/preferences';
+type EmailChannel = { email: boolean };
+type Channel = { [channel: string]: boolean };
+export type Preferences = { [category: string]: EmailChannel };
```

Comment:
I would move the categories here:
const categories = [
    'chat',
    'match',
    'course',
    'appointment',
    'survey',
    'news',
    'newsletter',
    'training',
    'events',
    'newsoffer',
    'request',
    'learnoffer',
    'alternativeoffer',
    'feedback',
];
and create a union out of it (`type Categories = typeof categories[number];`)  to use it instead of the `[category: string]`